string/global regex replacer/5719fd66-d597-45ee-bff8-64106b48879e.json

```diff
@@ -5,21 +5,25 @@
     "inputs": {
       "0e374f58-6b9c-453c-9d2e-e4887e0a367e": {
         "type": "string",
-        "name": "string"
+        "name": "string",
+        "description": "Receives the string to be replaced.\n\nExample:\n\n\"dog cat rain dog\""
       },
       "d1118ac0-749b-46e9-81af-8fc74dc14e10": {
         "type": "string",
-        "name": "regex"
+        "name": "regex",
+        "description": "Receives regular expression to test the specified string against.\n\nExample: \n\n\"dog\""
       },
       "3490f391-9b3b-441e-b4a2-b078669cf5b4": {
         "type": "string",
-        "name": "new"
+        "name": "new",
+        "description": "Receives the replacement string.\n\nExample:\n\n\"cat\""
```

Comment:
Can we use back references here? If yes, we should have an example for.

Reply:
I don't know wheather it is a good idea. Isn't it a javascript feature?